src/emissions/emission.py

```diff
@@ -92,39 +116,53 @@ def __init__(
         fuel_burn[1:] = fuel_mass[:-1] - fuel_mass[1:]
         self.fuel_burn_per_segment = fuel_burn
 
+        self.total_fuel_burn = 0.0
+        self.LTO_noProp = np.zeros(4)
+        self.LTO_no2Prop = np.zeros(4)
+        self.LTO_honoProp = np.zeros(4)
+
+    def compute_emissions(self):
```

Comment:
This method (and all the other emissions calculation methods) calculate the emissions and assign them to attributes of the `Emission` object. Why not instead have a method that calculates and *returns* the emissions? The approach you're using here hides what's going on: a user has to make an `Emission` value, then calls `compute_emissions` and then has to figure out which of the many attributes on the `Emission` value actually correspond to the calculated emissions. If you had an `EmissionCalculator` class instead (for example), with an `emit` method that returned an `Emission` object which was a data container containing the actual emissions data, things would be a lot clearer.